api/src/modules/characters/character.routes.ts

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+
+import { asyncRequestHandler } from '../../shared/async-request-handler';
+
+import { createNewCharacter } from './controllers/character.ctrl';
+
+export const characterRouter = Router();
+
+characterRouter.post(
+  'characters/create',
```

Comment:
paths should start from `/`, even if it works without them, real path have `/` on the beginning ;)